templates/about.html

```diff
@@ -35,7 +35,7 @@
       <p>At Cloudflare, we believe that open source, community-driven projects like CDNJS are the tools upon which the future of the internet will be built. Cloudflare is proud to provide the global CDN infrastructure that will help power that future.</p>
       <br />
       <a href="https://www.algolia.com/?utm_source=cdnjs&utm_medium=link&utm_campaign=cdnjs_about" target="_blank">
-        <img alt="algolia" style="margin-bottom: 20px;" src="/img/algolia256x80.png" width="192" height="60"/>
+        <img alt="algolia" style="margin-bottom: 20px;" src="/img/algolia.svg" width="192" height="60"/>
```

Comment:
It means that it will fill whatever space is available, so in this case the constraining factor is the height, so it doesn’t go over that.

As a result this looks the same as before. 

<img width="289" alt="screen shot 2017-04-22 at 13 13 18" src="https://cloud.githubusercontent.com/assets/6270048/25303959/9e017e1a-275d-11e7-839d-5468f6cff9f7.png">

<img width="608" alt="screen shot 2017-04-22 at 13 13 31" src="https://cloud.githubusercontent.com/assets/6270048/25303961/a3ad05e6-275d-11e7-8b48-9e871f139040.png">